evm_loader/lib/src/account_storage.rs

```diff
@@ -271,6 +271,7 @@ impl<'rpc, T: Rpc + BuildConfigSimulator> EmulatorAccountStorage<'rpc, T> {
         block_overrides: Option<BlockOverrides>,
         state_overrides: Option<AccountOverrides>,
         solana_overrides: Option<SolanaOverrides>,
+        tx_chain_id: Option<u64>,
```

Comment:
Why did you decide to use the transaction chain_id for overrides?
How could users create an override with a different chain id?

Reply:
it was requested by @s-medvedev here https://github.com/neonlabsorg/neon-evm/pull/400#discussion_r1593893772